climpred/tests/test_stats.py

```diff
@@ -150,26 +150,6 @@ def test_corr(control_3d_NA):
     np.allclose(actual, expected)
 
 
-def test_bootstrap_dpp_sig50_similar_dpp(control_3d_NA):
-    ds = control_3d_NA
-    bootstrap = 5
-    sig = 50
-    actual = dpp_threshold(ds, bootstrap=bootstrap, sig=sig).drop_vars('quantile')
-    expected = dpp(ds)
-    xr.testing.assert_allclose(actual, expected, atol=0.5, rtol=0.5)
-
-
-def test_bootstrap_vwmp_sig50_similar_vwmp(control_3d_NA):
-    ds = control_3d_NA
-    bootstrap = 5
-    sig = 50
-    actual = varweighted_mean_period_threshold(
-        ds, bootstrap=bootstrap, sig=sig
-    ).drop_vars('quantile')
-    expected = varweighted_mean_period(ds)
-    xr.testing.assert_allclose(actual, expected, atol=2, rtol=0.5)
-
-
```

Comment:
these two tests didnt make any sense. after random resampling of the inputs there should be no memory in the data.